packages/devextreme/js/__internal/ui/html_editor/matchers/m_wordLists.ts

```diff
@@ -21,6 +21,10 @@ function removeNewLineChar(operations) {
   newLineOperation.insert = newLineOperation.insert.trim();
 }
 
+function isTextOp(op): boolean {
```

Comment:
```suggestion
function isTextOperation(op): boolean {
```
But maybe we don't need this util at all? Actually even an object insert can be a "text operation" .. You basically check only that operation.insert is a string, maybe it's better to state it explicitly